src/main/java/AddList.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class AddList {
+    List<Mission> ls;
+    int counter = 0;
+    public AddList() {
+        ls = new ArrayList<Mission>();
+    }
+    public void printAllEvent() {
+        int counter = 1;
+        for(Mission a : ls) {
+            System.out.print(counter + ". ");
```

Comment:
Another way of writing output is to pass all the strings to Ui class and do unified formatting.